changelog/796.breaking.rst

```diff
@@ -0,0 +1,7 @@
+In ``plasmapy.particles``, rename the ``IonizationStates`` class to
+``IonizationStateCollection`` and replace the ``State`` namedtuple with
+the ``IonicFraction`` class.  When the ``IonizationState`` class is now
+provided with an ion, the ionic fraction for that ion is set to 100% for
+the corresponding element or isotope.  Renamed ``AtomicError`` to
+``ParticleError`` and ``MissingAtomicDataError`` to
+``MissingParticleDataError``.
```

Comment:
Use single back ticks `` ` `` so sphinx will auto-link the objects.

Reply:
I've been wondering about how we should handle links in the change log.  For example, past change logs may have references to `plasmapy.atomic`, though that subpackage doesn't exist anymore so the auto-links don't work.  I was putting them in double back ticks so that it would be future proof.  With that said...maybe it would still be better to do the auto-links regardless...though we'd need to remember to update the obsolete links in old changelogs.  I'll go with your suggestion.

Reply:
Update: I'm still using double back ticks for names for objects that no longer exist.

Reply:
Note: The single ticks `` ` `` are "future proof".  They will render just fine.  They just won't link to anything.

I don't think it's worth our effort to go through and manually change any of this for our old changelogs.  Being consistent about it will be a real pain.  People fully expect links to no longer exist for ancient changes.  If users really want the original references, they can go to the version specific documentation where the links are still valid.